pkgs/tools/networking/wait-for-it/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "wait-for-it";
```

Comment:
```suggestion
  pname = "wait-for-it";
  version = "unstable-2020-02-05";
```

Reply:
Agreed, but changed to
```
  pname = "wait-for-it-unstable";
  version = "2020-02-05";
```
so that it matches the Nix notion of derivation name/version:
```
nix-repl> builtins.parseDrvName "wait-for-it-unstable-2020-02-05"
{ name = "wait-for-it-unstable"; version = "2020-02-05"; }
```

Reply:
Would this affect say, `nix-shell -p`? I'd assume users what the least surprise here, and typing `nix-shell -p wait-for-it-unstable` is certainly more obscure than `nix-shell -p wait-for-it`.

Please let me know!

Reply:
`-p` uses attribute names so it behaves correctly. Only `nix-env` without `-A` will be problematic. I suggested using `yyyy-mm-dd-unstable` version format to sidestep this elsewhere.

Reply:
See also https://github.com/NixOS/nixpkgs/pull/88023#issuecomment-630326316